docs/user_guide/PXE_Boot.inc

```diff
@@ -217,7 +225,7 @@ various files to model different systems.
             kernel => 'OSTYPE-MAJORRELEASE-ARCH/vmlinuz',
             initrd => 'OSTYPE-MAJORRELEASE-ARCH/initrd.img',
             ks     => "https://KSSERVER/ks/pupclient_x86_64_el7.cfg",
-            extra  => "inst.noverifyssl ksdevice=bootif\nipappend 2"
+            extra  => "inst.noverifyssl ksdevice=bootif simp_disk_crypt\nipappend 2"
```

Comment:
This did not work as expected.   I was prompted for the passphrase (which I didn't know) after the client kickstarted.  For a KS client, is other setup required?

Reply:
@lnemsick-simp Does this work now with the fixes in https://github.com/simp/simp-core/pull/592? Looks like this was actually a long-standing bug.

Reply:
Yes, the bug fix addresses the problem.